source/release-notes/1.7.txt

```diff
@@ -0,0 +1,32 @@
+.. _c2c-release-notes-1.7:
+
+===============================
+Release Notes for mongosync 1.7
+===============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.7.0-c2c-release-notes:
+
+1.7.0 Release
+-------------
+
+**Upcoming**
+
+Time Series Collections
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting in ``mongosync`` 1.7.0, time series collections are
+synchronized between clusters.
+
+Minimum Supported Version
+-------------------------
+
+In 1.7.0, the minimum supported versions of MongoDB are 4.2.24, 4.4.23,
```

Comment:
Whoa. We publish the pre-v6 versions?

If we’re documenting our support for pre-v6 clusters, then we _should_ document the `timeSeriesHandling` parameter.

Reply:
@jason-price-mongodb Sorry for the confusion but we shouldn't be publishing pre-6.0 versions here. I updated the downstream release docs accordingly

Reply:
No problem. Removed versions per request. Cheers.